examples/python3/tripletex.py

```diff
@@ -29,8 +29,8 @@ def create_session_token(self):
         r = requests.put(f'{self.base_url}/token/session/:create', params=params)
         if (r.status_code == 200):
             return self.map(r)
-        else:
-            print(r.status_code, r.text, r.reason)
+
+        raise Exception(r.status_code, r.text, r.reason)
```

Comment:
Since this is getting converted in to throwing an exception the exception type it throws should be as accurate as practically possible. 
If none of the types [here](https://docs.python.org/3/library/exceptions.html#exception-hierarchy) fit then either a new custom exception type should be made by extending [Exception](https://docs.python.org/3/library/exceptions.html#Exception) or use [RuntimeError](https://docs.python.org/3/library/exceptions.html#RuntimeError).

There is a very informative answer on StackOverflow regarding this [here](https://stackoverflow.com/a/24065533/14996365)